src/services/message.service.ts

```diff
@@ -45,6 +47,7 @@ const sendMessageToDevice = async (
       },
       data: {
         notificationType,
+        ...context,
```

Comment:
What does the triple dot do here?

Reply:
That's the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax), which means here "put the key-value pairs from `context` into this object we're building for `data`.